service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapperTest.java

```diff
@@ -0,0 +1,128 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.time.ZoneOffset;
+import java.util.TimeZone;
+
+import uk.nhs.adaptors.gp2gp.common.service.FhirParseService;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+import uk.nhs.adaptors.gp2gp.ehr.exception.EhrMapperException;
+import uk.nhs.adaptors.gp2gp.utils.ResourceTestFileUtils;
+
+import org.hl7.fhir.dstu3.model.Observation;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ObservationStatementMapperTest {
+    @Mock
+    private RandomIdGeneratorService randomIdGeneratorService;
+
+    private static final String TEST_ID = "394559384658936";
+    private static final String INPUT_JSON_WITH_EFFECTIVE_DATE_TIME = "/ehr/mapper/observation/example-observation-resource-1.json";
+    private static final String INPUT_JSON_WITH_NULL_DATE_TIME = "/ehr/mapper/observation/example-observation-resource-2.json";
+    private static final String INPUT_JSON_WITH_EFFECTIVE_PERIOD = "/ehr/mapper/observation/example-observation-resource-3.json";
+    private static final String INPUT_JSON_WITH_ISSUED_ONLY = "/ehr/mapper/observation/example-observation-resource-4.json";
+    private static final String INPUT_JSON_WITH_NO_DATES = "/ehr/mapper/observation/example-observation-resource-5.json";
+    private static final String OUTPUT_XML_USES_EFFECTIVE_DATE_TIME = "/ehr/mapper/observation/expected-output-observation-statement-1.xml";
+    private static final String OUTPUT_XML_USES_UNK_DATE_TIME = "/ehr/mapper/observation/expected-output-observation-statement-2.xml";
+    private static final String OUTPUT_XML_USES_NESTED_COMPONENT = "/ehr/mapper/observation/expected-output-observation-statement-3.xml";
```

Comment:
I noticed that "/ehr/mapper/observation/" suffix is common for all the constants. Maybe we could move it to a separate one for a better readability?